jax/_src/numpy/ufuncs.py

```diff
@@ -2835,13 +2835,14 @@ def log10(x: ArrayLike, /) -> Array:
     [-2. -1.  0.  1.  2.  3.]
   """
   x, = promote_args_inexact("log10", x)
+  one_over_log10 = 0.4342944819032518  # exact value of 1 / log(10)
```

Comment:
nit: this would be somewhat simpler if you did
```
one_over_log10 = np.array(0.4342944819032518, dtype=dtypes.finfo(x.dtype).dtype)
```
then you could remove all the various `constant_like` calls below